source/_patterns/01-elements/input/input.scss

```diff
@@ -55,114 +165,193 @@
 	&[type="url"],
 	&[type="week"] {
-		// TODO: Evaluate whether those could get moved to ../_form-elements.scss
-		// Semitransparent is the default style
-		@at-root :where(#{&}) {
-			@extend %form-element-semitransparent;
-		}
-
-		// Floating labels
-		margin-top: to-rem($pxValue: 16);
+		@extend %db-body-sm;
 
 		&:not([type="date"]):not([type="datetime-local"]):not(
 				[type="month"]
 			):not([type="time"]):not([type="week"])
-			+ .elm-label {
-			// position: absolute;
-			// left: calc(#{to-em($pxValue: 10)} - 1px);
-
-			transition: opacity, transform 250ms;
+			+ label {
 			// Adopted by https://www.heise.de/developer/artikel/a11y-Reduced-Motion-4356171.html
 			@media (prefers-reduced-motion: reduce) {
 				transition-duration: 0.01s !important;
 			}
 		}
 
-		& + .elm-label {
-			color: $db-colors-secondary-enabled;
-			display: block;
-			font-size: to-rem($pxValue: 12);
-
-			margin-bottom: -#{to-rem($pxValue: 18)};
-			transform: translate(
-				$db-spacing-fixed-regular-m,
-				-#{to-rem($pxValue: 41)}
-			);
+		// Hiding inputs placeholder for floating label
+		&::placeholder {
+			opacity: 0;
+		}
 
-			& + .description {
-				margin-top: 0;
+		&:focus {
+			&::placeholder {
+				opacity: 0.5;
 			}
 		}
+	}
 
-		&[placeholder] {
-			&:placeholder-shown {
-				& + label {
-					opacity: 0;
-				}
-			}
+	&[type="search"] {
+		$paddingLeft: calc($db-spacing-fixed-sm * 2 + 0.625rem);
+		$icons-path: "../../../icons/" !default;
+
+		padding-inline-start: var(--elm-input-padding-inline-start);
+
+		&,
+		& + label {
+			--elm-input-padding-inline-start: #{calc(
+					$db-spacing-fixed-sm * 2 + 0.625rem
+				)};
 		}
 
-		&::placeholder {
-			color: $db-colors-secondary-enabled;
+		& ~ .elm-state-icon {
+			@include icon(glyph(search), 20, "filled", "before", false);
 		}
 
-		// TODO: Evaluate whether those could get moved to ../_form-elements.scss
-		// Validation
-		&:required {
-			&:valid:not([aria-invalid]) {
-				--formElement---borderColor: #{$db-colors-success-enabled};
-			}
+		+ label {
+			transform: translate($paddingLeft, calc(-1 * var(--db-sizing-md)));
+		}
 
-			&:invalid {
-				&:not(:placeholder-shown) {
-					--formElement---borderColor: #{$db-colors-primary-enabled};
+		~ .description {
+			padding-inline-start: $paddingLeft;
+		}
+
+		&:focus,
+		&:not(:placeholder-shown) {
+			&:not(:disabled) {
+				+ label {
+					padding-inline-start: 0;
 				}
 			}
 		}
 
-		&[aria-invalid="true"] {
-			--formElement---borderColor: #{$db-colors-primary-enabled};
+		// https://stackoverflow.com/questions/20804016/editing-input-type-search-pseudo-element-button-x
+		&::-webkit-search-cancel-button {
+			appearance: none;
+			position: absolute;
+			right: $db-spacing-fixed-sm;
+			top: 0;
+			height: $db-sizing-md;
+			width: 1em;
+			background: url(#{$icons-path}functional/images/navigation/db_ic_cancel_24.svg)
+				no-repeat 50% 50%;
+			background-size: contain;
+			opacity: 0;
+			pointer-events: none;
```

Comment:
why do we need to deactivate the `pointer-events` here? From my understanding this is the browser built-in element and that for I would expect that we actually need its functionality.
```suggestion

```